fusor-ember-cli/app/components/host-type-icon.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  didInsertElement() {
+    return Ember.$('[data-toggle="tooltip"]').tooltip({placement: 'right'});
```

Comment:
deleted host-type helper and created a component which allows us to insert the JS code for `.tooltip()`